core/menu.ts

```diff
@@ -387,27 +387,20 @@ export class Menu {
    *
    * @param e Key event to handle.
    */
-  private handleKeyEvent_(e: Event) {
-    if (!this.menuItems_.length) {
+  private handleKeyEvent(e: Event) {
+    if (!this.menuItems.length) {
       // Empty menu.
       return;
     }
-    // AnyDuringMigration because:  Property 'altKey' does not exist on type
-    // 'Event'. AnyDuringMigration because:  Property 'metaKey' does not exist
-    // on type 'Event'. AnyDuringMigration because:  Property 'ctrlKey' does not
-    // exist on type 'Event'. AnyDuringMigration because:  Property 'shiftKey'
-    // does not exist on type 'Event'.
-    if ((e as AnyDuringMigration).shiftKey ||
-        (e as AnyDuringMigration).ctrlKey ||
-        (e as AnyDuringMigration).metaKey || (e as AnyDuringMigration).altKey) {
+    const keyboardEvent = e as KeyboardEvent;
+    if (keyboardEvent.shiftKey || keyboardEvent.ctrlKey ||
+        keyboardEvent.metaKey || keyboardEvent.altKey) {
```

Comment:
I added a note to the PR description to mention this helpful removal of `anyDuringMigration`.